design/ceph-mon-pv.md

```diff
@@ -89,9 +89,53 @@ scheduling implementation in Rook. These restrictions will be removed or
 significantly relaxed once monitor scheduling is moved under control of Kubernetes
 itself (ongoing work).
 
-## Migration
-
-Monitor migration is currently not implemented. When nodes are decommissioned,
-for example, any monitors on the nodes are migrated using a fail over mechanism
-rather than also migrating the PVC. This support is currently being worked on in
-conjunction with the enhancements to scheduling.
+## Scheduling
+
+In previous versions of Rook the operator made explicit scheduling (placement)
```

Comment:
Was this code ever part of a release? I thought this was introduced during 1.1 cycle?
Shouldn't we include some of that text in the release note?

Reply:
Explicit monitor scheduling has been around for a long time. I reworked it to support AZ scheduling. I don't think that is in any release, but is in master as a backup plan.

I'll look at the release notes. I *think* they are written implementation agnostic, so may not need to be updated.